fdbserver/masterserver.actor.cpp

```diff
@@ -711,15 +711,10 @@ ACTOR Future<vector<Standalone<CommitTransactionRef>>> recruitEverything(Referen
 		TraceEvent("MasterRecoveryState", self->dbgid)
 		    .detail("StatusCode", RecoveryStatus::recruiting_transaction_servers)
 		    .detail("Status", RecoveryStatus::names[RecoveryStatus::recruiting_transaction_servers])
-		    .detail("RequiredTLogs", self->configuration.tLogReplicationFactor)
```

Comment:
Why are you dropping this info from this trace event?

Reply:
They are included in self->configuration. As we are logging the whole self->configuration now, we can deduplicate them?

Reply:
I think so. Maybe @sfc-gh-ajbeamon has an opinion (as a consumer of these metrics) on this. I will let you and @sfc-gh-ajbeamon figure this out.

Reply:
I think that sounds reasonable, assuming `self->configuration.toString()` does include all of these fields being pulled from the configuration before